### ISSUE TITLE
Remapping Defenses

### ISSUE THREAD

**Issue by vehemot** [openlegend/core-rules#39]
Suggestions:
The first suggestion I have is related to the physical stats. I recommend separating Agility 
into two stats, Agility and Precision. This is due to the fact that Agility, as it is now, is 
simply a better stat than Might. It contributes to both offense and defense, while Might only 
contributes to offense. By splitting this stat into Agility and Precision you can make Agility 
the defensive stat and Precision can be the offensive stat. Additionally, the two stats still 
have plenty of secondary skill contributions  (E.G. Agility: dodge attacks, stealth, perform 
acrobatics, escape a grapple. Precision: shoot a bow, pick a pocket/lock, perform sleight of 
hand, juggle). Another option is to combine Might and Fortitude into one stat to match the competitiveness of Agility, but it makes more sense to me to split Agility.

My second suggestion is related to the first suggestion. I recommend having agility, 
presence, and will contribute to hit points but to a lesser degree than fortitude. This is a 
fairly simple change but prevents some scenarios where your caster has more hit points than 
your front liner while allowing your defensive stats to still contribute to your hit point 
total. My suggested HP contribution is as follows:

Hit Points = 3(Fortitude)+ 2(Presence) + (Agility+Will)+10 

The reason being that Fortitude is a major direct contributor to your vitality 
whereas Agility allows you to efficiently mitigate damage allowing you to fight longer while 
avoiding damage. While Agility is not increasing your hit points directly it does allow you to 
fight longer and should have some contribution to HP. The same holds true for Presence and Will 
which are mental contributors, and therefore indirect contributors, to resisting damage by 
overcoming damage and pushing your body past its limits. Presence, however, contributes to both mental and physical willpower (see my reasoning for this below my third suggestion) giving it two indirect hit point contributions and as such should contribute slightly more than will and agility which only have one indirect contribution.

My third suggestion is that you modify your defense calculations to the following:
Toughness = 10 + Fortitude + Presence
Evasion = 10 + Agility + Armor
Resolve = 10 + Presence + Will
By moving to this strategy you allow Evasion to be your defense stat for getting out of the way 
of negative effects including weapon attacks and some spell attacks like a death ray, firebolt, 
or fireball where you can intervene your armor/shield between you and the effect or get out of 
the way of the attack completely. Your Toughness then becomes how well you resist effects that 
target your constitution that are hard/impossible to dodge and need to get their hooks into 
you. For example things like a chill wind that would freeze you, a necromantic essence drain 
that would sap your vitality, or a poison fog that threatens to suffocate you and corrupt your 
body. Finally, this leaves Resolve as a defense against mental attacks that have the same 
nature as the examples for Toughness but with respect to the psyche. Meaning that they are hard 
to avoid but need to get their hooks into your mind before becoming a threat. For example 
attacks such as a banshee's wail, a bard's cutting words, a lethal phantasm, or malefic 
visions.
In defense of presence contributing to both Toughness and Resolve. I see social skills as a 
mixture of mental and physical contributors that manifest into a new third category. This 
supports the idea that Presence is not only your physical willpower but your mental force of 
personality. Using this approach to the stat you can see why it would contribute to both 
physical and mental defenses to stave off harmful effects.

My fourth suggestion is that you clarify how area of effect attacks accrue disadvantage. The 
current wording isn't entirely clear as it seems to suggest that one 5x5x5 cube would impose 
disadvantage 3 because it has three 5' lengths. This doesn't seem to be RAI and you might 
consider changing the wording to 1 per 5 cubic feet or per 5' cube depending on what the intent 
is. For disadvantage 2 at 10x10x10 go with the former, for 5x5x10 the latter. I would suggest 5 
cubic feet to make it simpler, and to give players a reason to use area of effect attacks 
rather than multi-attacks for hitting multiple targets, which I feel is your intent. 

Other than these suggestions I have mostly nit picky naming consistency, and other syntax/wording. I want to say that I really like what I have seen here and would be very interested in playing the game. I believe that the suggestions I have made above would improve what is an already amazing rule set. Thanks for all of your hard work towards breaking down genre limitations and commitment to the community. I will probably continue to contribute as I analyze the rest of your rules in the form of editing oversight if I feel that my input is desired.

**Comment by brianfeister** [comment]
@vehemot Well all input is welcome, even if we can't accommodate it all 😄 

All of the proposals you make here have the potential of being HIGHLY disruptive to the existing community, so we will need to tread lightly because of that.

1. When I first read the suggestion around breaking Agility into two, I cringe. ALOT. Because of how big the impact is. And while I was initially intending to respond and basically say "Nah, we won't do that", I find myself considering it. While this game isn't meant for hardcore literalists and munchkins, if fixing this is something that will help them enjoy the game, we should at least entertain the idea. @istabosz your thoughts on bullet point 1 above?

1. For changing the HP formula, I don't disagree with your objection about the potentially unbalanced high HP wizard, your proposal actually matches something I had before but moved away from. I think for this one I'm comfortable just saying "Nope." ... while it fixes a realism problem it creates a bigger problem than it solves by making your HP formula very confusing for the less analytically minded. If you want to propose a solution that is much closer in complexity to the original formula, I'd consider it.

1. Changing defenses is interesting but it's almost the opposite of your suggestion in point 1. There, you're striving for balance by breaking up the usefulness of Agility. Here, you're making Evasion **excessively** useful by funneling all melee attacks toward one defense instead of splitting among two. Balancing out combat between fighting styles is the reason it is the way it is now. Your version of Toughness would be a very rarely utilized defense. We already have one of those with Resolve, but it's offset by the lack of armor benefit. I'll tell you what I'd be more likely to consider, since this question of which attack targets which defense and Precise vs. Forceful weapons keeps coming up. I would be more willing to consider one defense score and all three of Fortitude + Agility + Armor factored in. Because then we're just saying it's a combines representation of : 1) How hard of a punch you can take / fireball you can withstand without being taken out + 2) How good you are at getting out of the way + 3) How much armor protection you have.... the more I think about it the more I like it.... We would sort of be resurrecting "AC" with this, which I'm not terribly afraid of, though I think we'll name it something other than AC, just for fun... It's worth noting that I often forget to hold players to the restrictions around Precise vs. Forceful weapons and their eligible defense targets, so this would clean things up with the game and we could otherwise accept your suggestion, having "Toughness" / "???" / "Resolve"....  Thoughts @istabosz? I think the only reason this hasn't been on my radar is because I accidentally brush it under the rug when I run the game and have come to do so subconsciously.

1. Point 4 seems a simple editing suggestion, I'm going to leave @istabosz to decide if he want to modify that explanation or not. 

Glad you've got a high opinion of the game @vehemot ... and I'm shocked that I'm wiling to consider such drastic changes so late in the development cycle. We'll see what happens.

**Comment by DMsShadow** [comment]
The agility might thing reminds me of the old Str vs Dec and finesse fighting in 3.5. While agility and precision is making me think of Skills and Powers! I want to sit down (if the many kids ever give me a chance) and go through it properly, but my initial thought is that it doesn't come up too much due to the Toughness/Evasion split. Essentially if most of the players focussed on agility, they'd get hit by lots of agile ninjas with high evasion, lower toughness. Therefore rewarding anyone who diversified into might.

**Comment by brianfeister** [comment]
@DMsShadow Thanks for the input - we're typing at the same time. I'm curious your reaction to my above response. Lots of intense mechanical considerations here.

**Comment by brianfeister** [comment]
Bullet point 3 (with my modification added) does make the game alot simpler since most all attacks will now target one defense and that does reduce the cognitive burden. I like the idea of just having Fortitude and Agility stack on the exact same defense, but the problem becomes doing that without ruining armor. 3 potential 9 point bumps to that defense is bad news for the potential 27 defense score. But maybe that actually does work as I muse on it more... hmmm.... because right now, the damage output is beating the defense bump so this might be a good correction. Hmmm.... 

I always figure "epic max damage output" to be around d20+3d10 Advantage 5... in reality it could be higher, but most characters won't be that narrow and will diversify some. 

So, the average roll for d20+8d10 keep 3 = 40.68 ( @vehemot  you seem like you're probably a number person, so you might like [this open legend roll average calculator](http://spandox.com/rollopen.htm).

So, if 40.68 is being pitted against a defense of 27 for a (WHAT ARCHETYPE WOULD EVEN DO THIS) Agility 9 + Fortitude 9 + Armor 9 then I suppose that works. 

Actually, we might be totally fine on balance here. We can continue to put a handicap on Agility via Armor max Agility bonus. Then, everything can stay as is and you can't abuse Agility to get a 27 "AC" because you'd be wearing full-plate with a max agility bonus of 5 or something

**Comment by brianfeister** [comment]
It sure does make targeting someone's "non-AC" defense super powerful, but I suppose it's not a real change, given that Resolve-targeting effects are already CRAZY powerful as well since Armor can't defend against it.

**Comment by WarthogRoadkil** [comment]
Splitting up Agility seemed pedantic at first, but I think the comparison to Fortitude-Might is a valid one. The only concern I have is the possible bloat in adding another Attribute. @brianfeister can correct me on this, but there's already quite a few Attributes, and adding one more doesn't seem like a big deal, but there comes a point where players are going to be spreading their points too thin across too many attributes, and that's a balancing concern I would ask Brian to comment on.

**Comment by brianfeister** [comment]
@WarthogRoadkil the conclusion I gradually worked my way toward was that actually, if we make a new defense that is "Agility + Fortitude + Armor" but make it so that Armor and Agility limit each other (typical in many systems), then that means that Agility is no longer overpowered because only Fortitude can combine for MAX adjustment to the defense score / AC.

**Comment by Groumy** [comment]
I have a problem with the Fortitude+Agility+Armor defense and it's the relation with the HP.

HP are composed of Fortitude, Will and Presence. If we do that trifecta defense, Fortitude is now represented twice in both _evading_ damage and _sustaining_ it.

It might be OK to do so, maybe that's the way to counter the fact that agility is used for both attack and defense with Fortitude now used twice for defense. We just have to make sure that it his desired.

With that said I think that presence would make more sense in the defense as in if you have an intimidating presence that could affect the opponents in their attack against you.

Maybe HP could be Fortitude+Will x4 and Defense be Agility; Presence;Armor ?!? Just throwing idea here and challenging conventions don't throw stones at me ;)

**Comment by vehemot** [comment]
@brianfeister Glad to see that my recommendations could prove useful, even if it merely inspired an improvement ,it is quite humbling. I have some follow up thoughts on a few points and here they are.

2. Perhaps **Hp = 3xFort + 2xPresence + Will + 10** would  approach a happy middle ground. I do understand that 2(Fort + Presence + Will)+10 is simpler, in that it has fewer steps, but added realism can inspire immersion so it may be worth the sacrifice of adding in two steps. 

3. Thinking about it now I begin to wonder whether it would be better to make one defense for all damaging attacks and one defense for all bane resistance. Correct me if I am wrong, but I think this is very much in line with your current considerations if not identical to them.

4. You are correct on that. The change of saying 'per 5 cubic feet' instead of ' per 5' of length' would be fairly simple and would add clarity.
 
@DMsShadow The reason I proposed this is that when you put points into Agility you get a lot of bang for your buck making it a natural favorite over might. This is because you get offense and defense out of it. This leaves you with more excess points after finishing your core build than one would have if they went with a Might build as Might builds would then have to spend points on defense in addition to their attack attribute. The same is true for casters, but they get some swanky stuff to supplement them. That means that someone can put points in Agility, Fort, Presence, and Will fairly easily, making them quite tanky while having comparable damage output to any other member of the party. When compared to their Might based compatriot they would likely be substantially tankier with greater utility, especially with armor as it is. 

But if @brianfeister implements a Might dependence on wearing armor or restricts Agility's contribution when armored, or both, this problem could be heavily mitigated by giving Might users a non stat method of armoring up to make up for this shortcoming. Given that @brianfeister has just stated he is considering at least one of these options as I type this, it may prove unnecessary to split them up.

**Comment by brianfeister** [comment]
@istabosz waiting for you to weigh in on this one 😄

**Comment by istabosz** [comment]
So, correct me if I'm wrong, but the proposal is to have the following 3 defenses:

Toughness = 10 + Armor + Fortitude

Armor Class (or whatever) = Armor + Fortitude + Agility

Resolve = 10 + Will + Presence

My concern with the new AC defense is that it doesn't have that 10 point buffer, which means heavy casters could end up with an AC of like 2. If we did add a 10 point buffer, it means that people could twink out an AC of 37.

I did really like @vehemot 's original suggestion of:
Toughness = 10 + Presence + Fortitude
Evasion = 10 + Agility + Armor
Resolve = 10 + Will + Presence

I don't think that would under power the Toughness stat, as 16 out of 29 banes target toughness. And, if we add an agility cap to heavier armors, that could prevent Agility from becoming overpowered. The real reason I like this suggestion is because it means that armor is no longer protecting you from things like poison gas or necromantic life drain AND it solves the confusion over weapon attacks targeting different defenses.

**Comment by brianfeister** [comment]
Oh no, I still plan to have the 10 point buffer! The musing I did above was around the ceiling of the defense score with 3 potential bonuses (armor, agility, fortitude). It would still have the base of 10.

**Comment by brianfeister** [comment]
Agreed fully on everything. My proposal was this:

Toughness = 10 + Presence + Fortitude
{???} = 10 + Fortitude + Agility + Armor
Resolve = 10 + Will + Presence

And alot of things that target Toughness would maybe move to target {???}

Note that Fortitude play in twice, making it more useful than it is now, which I believe is correct given it's competition with Agility.

Yup, I think we actually are in full agreement. I can't wait to apologize to the community for this, that'll be fun.

**Comment by brianfeister** [comment]
Re: AC ceiling, cap Agility + Armor at 5 and you'll have a 33 max. Given the progression toward an average roll of 41 for a high level attack, I thought it might be fine.

**Comment by istabosz** [comment]
Ahh. Gotcha. I like it. Now the really tough part... naming it.

My first impulse was "Protection", then I remembered that's what we call Abjuration now.

**Comment by brianfeister** [comment]
Same for me, I thought "Protection" first. I really don't want to rename that attribute (that was just renamed) to save it from this change.  So weird because AC is so prevalent, and while the abstraction is fine, the name is totally innaccurate.

**Comment by WarthogRoadkil** [comment]
I think it's going to be tough finding a name for it that won't just be a synonym for Toughness. Evasion I think still fits. It's like the bob-and-weave, even if you still get hit, you've successfully mitigated a % of the strike by slipping out of the way.

**Comment by vehemot** [comment]
A couple of options Combat/Martial/Defensive Defense/Prowess/Class (These are meant to be interchangeable), Resistance.

**Comment by DukeAnax** [comment]
I also think just keeping it at Evasion would work, You avoide the blow because you dodge and/or it glances off your armor.

Also what do you generally think about the idea that in this new system Attacks target Evasion, but Damage targets Toughness somehow?

And Brian, you don#t have to appologise to anyone for anything. DnD pulls stuff liek this every edition, this is still in progress on the first.

**Comment by Groumy** [comment]
How about resilience ?!

**Comment by brianfeister** [comment]
Resilience isn't bad. Also Avoidance is a thought

**Comment by DMsShadow** [comment]
1. To get back into this after my weekend off herding children. @vehemot I do understand the thinking around splitting agility up (and I know this is a bit defunct now with the other ideas), I'm just not hugely keen on it. 

2. I'm kind of ambivalent on the tinkering with the HP equation. I tend to use programs to autocalc it so the complexity doesn't often bother me. However, from a design principle, simple is always better. I'm also not convinced on the making fort stronger argument, as you could argue that many people who are badly injured go on because of 'force of will' alone. Is it that you have a body developed to resist pain, or that you have a force of will to power past it? So I guess I'm arguing here to back up @brianfeister statement to keep it as is.

3. I'm really enjoying the debate on these. I like the idea proposed by @brianfeister of:

> Toughness = 10 + Presence + Fortitude
> {???} = 10 + Fortitude + Agility + Armor
> Resolve = 10 + Will + Presence

As it'd bee good to see more parity around the derivation of these values. Our new Evasion value at first seems a bit buffed up due to the 3 stats instead of two, but if that limit around Max Agility or Min Might for armour comes in to play it should balance nicely (as effectively there's some trade off that will limit at least one of these. 

I'd be keen to stick to the name of **Evasion** for a few reasons. For one it'll cause less disruption to the larger community as you're not changing the stats around (technically), and I agree that toughness sounds more like a sit there are take it type thing, while the other should be about deflection, dodging, rolling with the punches etc.

The key will be splitting attack types up enough that there are advantages and disadvantages to having them each higher and lower.

**Comment by Elvenoob** [comment]
I agree with leaving it as Evasion, and, as for why Fortitude is relevant to Evasion, well, think of it as more than just a single dodged bullet.

Over the course of an adventurer's day, you're gonna have a good, large number of attacks targeting your Evasion, and whilst dodging one doesn't seem like a feat of endurance, dodging 10 or 20 such attacks is gonna need at least a bit of stamina.

**Comment by istabosz** [comment]
I'm good with calling it Evasion too. However, I've been pondering why we need Fortitude linked to two defenses. Is it really underpowered? If we use these formulas:

Toughness = 10 + Presence + Fortitude
Evasion = 10 + Agility + Armor
Resolve = 10 + Will + Presence

Then Agility and Fortitude compete as follows:
**Fortitude contributes to...**

- Hit Points
- Toughness
- Armor prerequisites

**Agility contributes to...**

- Attacks
- Evasion
- Dexterity related action rolls (acrobatics and such)

That seems pretty balanced to me.

**Comment by Elvenoob** [comment]
I think the original problem was *might* being underpowered, and we got sidetracked with re-calculating armor values somehow...

**Comment by DMsShadow** [comment]
The original comment was around Agility being overpowered, making Might less useful by comparison. 

Part of fixing that was to perhaps change the defences relating to these and to limit the influence of Agility on Evasion etc.

**Comment by Elvenoob** [comment]
Hmmm... I guess that makes sense.

**Comment by DukeAnax** [comment]
@istabosz The idea was that Evasion becomes the main defense and Toughness more specific. So that Agility could have less relative power, whithout reducing the overall power of Evasion.

But now that you mention it, it sounds a bit weird that Fortitude would contribute to avoiding the blow.

However if we distribute it like istaboz suggested, while still giving more defensive cabability to Fortitude. How about my suggestion of simply using both defenses together?

First the Attack Roll would be mesured against Evasion to determine Hit and Bane, but on hit you'd take the Attack against Toughness to determine damage?

That way a nimble character would be hit less, but when they do, they get hit harder, while a Tough guy can soak up the hits.

**Comment by istabosz** [comment]
@DukeAnax A noble suggestion, but I think it goes against one of the goals of simplification of the system. One thing that people have repeatedly applauded is the single attack roll mechanic. And, while your proposal doesn't add an additional roll, it does add an additional calculation that might potentially slow combat down even more than damage rolls do.

**Comment by DukeAnax** [comment]
understood, makes sense

In that regards, the unified main defense that brianfeister suggested makes a lot of sense too.

**Comment by DukeAnax** [comment]
Would you guys willing to rename the Supernatural Attribute again and call it so0mething like "Blessing" So that Evasion can be renamed Protection?

I found it a bit weird that "Protection" would bind Demons anyway.

**Comment by Elvenoob** [comment]
Wait, what? Most people are fine with it staying as Evasion...

**Comment by DukeAnax** [comment]
I was too, just a different thought.

**Comment by istabosz** [comment]
Well, if the problem is that Might is underpowered (which I think I agree with), I think we should find a way to balance that. Even if we manage to balance Agility and Fortitude, that still leaves Might at the bottom of the pecking order.

**Comment by DukeAnax** [comment]
How about putting Might into Tougness?

**Comment by brianfeister** [comment]
I may be OK with leaving `Might` as the odd statistic out. I'm still thinking about that.

Originally, the goal was to balance Might and Agility, however, from my perspective that shifted because I decided that the primary goal of this exercise was to fix the problem where `Forceful` vs. `Precise` attacks target different defenses. The idea of an unnecessary straddling that divides "AC" has created lots of confusion around "well if I use a Forceful / Precise weapon, what do I do?" That's the main thing I want to fix. The existing way has received alot of push back from many people.

**Comment by DMsShadow** [comment]
Did we mood the point of making Might a prerequisite for armour? That way you choose if you want a high Might to get access to big forceful weapons and then use higher grade armour, or you focus on Agility to dance around enemies but are only really good with precise weapons.

**Comment by DukeAnax** [comment]
Well that's another point, right now there are no "BIG forcefull weapons". except for using a different attribute, they are essentially the same as finesse weapons. They don't do any more damage for example.

**Comment by DukeAnax** [comment]
The thread has once again gotten a bit hard to follow, so let me try to recap and tell me where I'm wrong:

All weapon and projectile attacks will now target Evasion, while Toughness only concerns effects that directly affect the body, like poison or area affects, simmilar to how Resolve protects against mental effects. (I really like the simplification and the symmetry there, well done)

**Resolve**: 10 + Will + Presence
Remains the same as before

**Toughness**: 10 + Presence + Fortitude
Why exactly is Presense in there? Wouldn't Will be more appropriate if anything? Also Might is generally associated with Toughness and it would help with the issue of Might being underpowered.

**Evasion**: 10 + Agility + Armor (+ Fortitude)
There currently is a discussion whether Fortitude should be included here. As istabosz pointed out: it already contributes to defense through HP. However the same is true with Toughness and with Agility and Armor planned to be somewhat mutually exclusive, Fortitude may be needed here to achive sufficient values. it also makes sense that you need endurance to keep dodging.

**Armor** will limit Agility. it wasn't clear wether heavy Armor will require, Might, Fortitude or both.

**Comment by brianfeister** [comment]
@DMsShadow My thought is that we will use 

Yeah. Lots of people's voices in here and tough to sort through. I think **Might** for Armor pre-req might be the right direction @DMsShadow. I'm leaning that way. That would give us:

* **Fortitude** affects HP + "Evasion" + Toughness <--- I'm OK with 3 here because it doesn't affect * * **both** defense and offense
* **Might** affects attacks + Armor quality
* **Agility** affects attacks + "Evasion"


Then Fortitude can be the attribute that  HP + the 3rd modifier for "Evasion" if that's what we call it. That name still leaves a bad taste in my mouth though.

**Comment by Groumy** [comment]
If the fact that different type of attack target different defenses is still an issue (not quite sure with the back and forth of everybody, but I'm still not comfortable with the idea that you just can't dodge that two-handed sword attack) how about to simply, let the defendant choose how he react to the attack and pick the defense he wants ...

So a character with a high Toughness will simply try to soak it, and the character with high Evasion will try to avoid it (if able to, of course).

Just sayin'

**Comment by WarthogRoadkil** [comment]
That entirely defeats the purpose of multiple defenses, though.

**Comment by brianfeister** [comment]
That's not quite the idea here. My most recent post is like this:

**Toughness** = 10 + Presence + Fortitude
**{???}** = 10 + Fortitude + Agility + Armor
**Resolve** = 10 + Will + Presence

And has the followign impact on game balance:

* **Fortitude** affects HP + "Evasion" + Toughness <--- I'm OK with 3 here because it doesn't affect *both* defense and offense
* **Might** affects attacks + Armor quality
* **Agility** affects attacks + "Evasion"

I think having the defender choose breaks the rule of simplicity so I'm thinking I won't go that dierction.

And to remind everyone again, the goal here is to SIMPLIFY because right now Agility and Might attacks can be confusing because they switch between different target defenses. The goal is to streamline and push all Agility and Might attacks toward **{???}** (with **Evasion** currently in the lead for it's name)

**Comment by brianfeister** [comment]
Yeah. Lots of people's voices in here and tough to sort through. I think
Might for Armor pre-req might be the right direction. Then Fortitude can be
HP + the 3rd modifier for "Evasion" if that's what we call it. That name
still leaves a bad taste in my mouth though.

**Comment by DukeAnax** [comment]
I like your proposal of Resilience. It fits all three factors of Fortitude, Agility and Armor better than Evasion does.

**Comment by brianfeister** [comment]
I like Resilience too... how do others feel about it?

**Comment by WarthogRoadkil** [comment]
So, a synonym for Toughness? That could be confusing.

**Comment by DMsShadow** [comment]
@WaryhogRoadkil summed up my concerns over it. Sounds to similar otherwise I'd be happy with it. Just struggling to come up with anything else. I was even interrogating Thesaurus.com for ideas.

**Comment by DukeAnax** [comment]
Might be because I'm not native, but I don't see the issue. Resilience implies flexibility, so is almost the opposite of Toughness. Also I can't find a Thesaurus that links the two. I often found a link between oughness and Resolve though...

Otherwise, how about renaming Toughness as well? Maybe Tenacity?

**Comment by Elvenoob** [comment]
I'd still prefer leaving it as Evasion, it's less confusing, and still fits, as I mentioned quite a while ago, dodging one thing might be easy, but you're gonna be evading a lot more than just one attack over the course of an adventuring day, and that'd be tiring, which is where the Fortitude fits in.

**Comment by DukeAnax** [comment]
One might argue though that a new name is less confusing, becasue it reminds people that there has been a change and to not use the old version.

**Comment by brianfeister** [comment]
![screenshot 2016-10-27 15 22 50](https://cloud.githubusercontent.com/assets/806536/19781826/43ba895e-9c59-11e6-823f-a57d26d05153.png)

**Comment by brianfeister** [comment]
I agree @DukeAnax a new name might help to highlight the change, I'm inclined to lean that direction, but also we shouldn't make a choice now that impacts the game for years to come because of a temporary / outdated rule.

**Comment by istabosz** [comment]
I would also like to point out that changing the name of Evasion means it has to be updated everywhere.

**Comment by istabosz** [comment]
Woops. Mis-click :-)

**Comment by raihje** [comment]
You could just name the trait responsible for all defensive purposes "Defense" and then you have your raw toughness which dictates how heavy your armour can be, and contributes to HP and resisting various body-targeting effects (Poison, etc) and Resolve which is mental reserve.
I'm rather fond of the one-defense idea even if it does take some of the depth out.

I know the goal here is simplicity, but I'd remind those of you in charge of designing that simplicity often comes at the cost of depth. This particular change sounds great though.

**Comment by brianfeister** [comment]
Just a reminder that we're only looking for name suggestions for **{???}** right now... and possibly **Toughness** as well

Here's the current proposal:

**Toughness** = 10 + Presence + Fortitude
**{???}** = 10 + Fortitude + Agility + Armor
**Resolve** = 10 + Will + Presence

And has the following impact on game balance:

* **Fortitude** affects HP + "Evasion" + Toughness <--- I'm OK with 3 here because it doesn't affect *both* defense and offense
* **Might** affects attacks + Armor quality
* **Agility** affects attacks + "Evasion"

All non-supernatural attacks will target **{???}**

And to remind everyone again, the goal here is to SIMPLIFY because right now Agility and Might attacks can be confusing because they switch between different target defenses.

**Comment by DukeAnax** [comment]
As far as I can tell, the only viable option we have so far are:
Keep them as Evasion and Toughness
Change them to Resilience and Tenacity
The other options conflict with and would require to rename other parts.

Also clarification please: Do you plan to _change_ armor from Fortitude to Might or do you want to use both? Both would make sense,  as you need a certain Might to move in them, but Fortitude to wear them over periods of time.

**Comment by Groumy** [comment]
Suggesting "Physical" as in : you attack the Physical Defense with weapons and some supernatural attacks.

"Physical" kind of naturally include Fortitude and Agility and Armor so the change will be easy to remember to old timers and will be also easy to grasp to new comers. Also I think it will still make sense when the system will have a couple of years.

But that makes Toughness kind of out of place (should be included in a Physical defense). 

Tryin' to help you have a spark of genius here ;)

**Comment by DMsShadow** [comment]
Just going to throw a few out there, not so convinced by them though:

Defense (already a generic term in use)
Protection (Already an ability)
Deflection (may not fit as well with fortitude)
Guard/Guarding (meh)
Ward/warding
Divert/Diversion
Repulsion/Repel
Block/Blocking

That's what happens when you trawl a thesaurus.

**Comment by jjnich** [comment]
Aegis = 10 + Fortitude + Agility + Armor
Warding = 10 + Presence + Fortitude
Resolve = 10 + Will + Presence

My suggestions above. Aegis would be defense against all physical attacks, Warding would be defense against AoE and some supernatural attacks, Resolve would be defense against mental type attacks I guess.

I'm not even totally sold on these.

Has it been considered to just have 2 defenses? example below
Toughness = 10+Fortitude+Agility+Armor (defense against physical and AoE attacks)
Resolve = 10+Will+Presence+Prescience (defense against non-AoE supernatural attacks)

**Comment by DMsShadow** [comment]
I thought about Aegis for a bit, but it feels to obscure and its meaning is more about being under the protection of someone/something else so not sure it fits. Depends on what Brian and Ish feel though.

**Comment by istabosz** [comment]
Right now, my favs are Toughness, Defense, and Resolve.

Since our Defense stat is really just the OL equivalent of Armor Class, I think Defense is actually a pretty good fit.

**Comment by DukeAnax** [comment]
The problem with that is that Character Creation reads: "Calculate your defenses: Toughness, Defense and Resolve"

**Comment by vehemot** [comment]
Changes of course must propagate if something is competing for the same name. For example: Calculate your resistances: Toughness, Defense, and Resolve. Given that Toughness and Resolve resist bane effects and Defense would resist damage that seems appropriate to me. I also agree that Defense is accurate for the purpose of that resistance stat.

**Comment by Groumy** [comment]
I guess we are waiting on @brianfeister for the final say of the name ? Or do you want to get the pulse from the community through a poll on the mightybell community ?

**Comment by raihje** [comment]
I could implement this on my fork and if he wants he can pull it or wait to decide a name on mightybell.

**Comment by vehemot** [comment]
It occurs to me that if you change the HP formula to **Hp= 2x(Will + Presence + 2xFort) + 10** and change the Toughness dependence on armor to a Might dependence you can kill two birds with two stones by shifting some value to the Might stat while making sure the value of Toughness isn't reduced.

**Comment by Hvalfisk** [comment]
I like the symmetry of this:

Toughness = 10 + Fortitude + Might + Presence
Defense = 10 + Fortitude + Agility + Armor
Resolve = 10 + Fortitude + Will + Presence

Fortitude is the main defensive stat, so it adds to all resistances. I think it makes a lot of sense for fortitude to help defend against mental attacks. 
Might is the physical power stat, so it adds to Toughness. 
Agility is the dodging stat, so it adds to Defense.
Will is the mental defensive stat, so it adds to Resolve.
Presence is a kind of external Will/Might, so it adds to both Toughness and Resolve.

**Comment by brianfeister** [comment]
Just arrived in Florida for vacation. Need time to process all of this.

**Comment by brianfeister** [comment]
(I've unlocked the comments on this issue)

I was all set to sit down, make a hard decision about this and begin implementing the rules text changes, however, **no such luck!**

The problem that I'm seeing is that this is accidentally giving a special value to **Might** by removing it's "peer" **Evasion**. In doing so, we're saying that our new proposed stat **Defense** (which I like but is bothering me because it hurts our ability to use "Defenses" as a category) for some reason incorporates both **Fortitude** and **Agility** but for some reason there's another Fortitude-based defense. That just doesn't sit well with me.

How do folks on this thread feel about just dropping **Toughness** and having only two defenses:

**Resilience** (my proposal for the **Fortitude** + **Agility** combined defense)
**Resolve**

I kind of like the funny "R&R" and the word similarity, while it might not be perfect. I guess I just like it better than saying **Defense** and then losing the ability to categorically call the two "Defenses"

**Comment by WarthogRoadkil** [comment]
I think fewer defenses is nice. Less looking at rules, more gaming.

**Comment by brianfeister** [comment]
By allowing both **Agility** and also **Fortitude** to contribute to the main defense used for most all attacks, it also helps out with another problem - right now it's a little bit difficult to make a character with decent Defense scores because typically the high stat will go to your primary attack attribute. This leaves you sometimes struggling with a 2 or 3 in defensive stats. And because that resulting 12 or 13 defense score is going to cause you to take **alot** of extra damage, it's a bit problematic.

I think if we go this direction I'm also going to revamp armor to have a much smaller range in numeric value. I've dealt with the full-plate wearing tank characters and the way it works is fairly predictable - they basically take no damage until you get them in a group situation and you can attack them with many adversaries (like 4 - 10) each turn so that one of the adversaries gets a huge dice explosion. Outside of that they're kind of unkillable and unstoppable.

**Comment by jjnich** [comment]
I like the 2 defense options (as I suggested something similar earlier) I also really like the names you chose for them.

**Comment by Groumy** [comment]
I also like it a lot. It's way simpler to have a physical defense (Resilience) and a mental one (Resolve).

I wonder though if we will miss the pure Toughness one in case of poisons and diseases for example. It feels odd to have agility contribute to your resistance to poison.

That could be fixed with poisons/diseases having a CR and requiring a kind of Fortitude Saving Throw, or an opposed roll.

I just don't want use to loose that perspective.

**Comment by jjnich** [comment]
Have a fort action to resist a poison bane is easy enough, but what happens when you fail? Success with a twist? 5 damage to you and the enemy? I like the idea, just bringing up an issue I had playing this weekend when it seemed like the "something always happens when you roll" rule was hard to deal with. But I'll be putting more detail on that in another thread.

**Comment by Hvalfisk** [comment]
You could argue that higher agility allows you to more quickly cover your eyes/mouth/face, thus protecting you against poison clouds, firery explosions, other area attacks.

**Comment by Groumy** [comment]
I wasn't thinking about the "infection attack" @hvalfisk. But that could be it.

Infecting attacks could target Resilience but if the poison/disease require more check to see if it's spreading that could be handled through Fortitude checks.

**Comment by DukeAnax** [comment]
On one hand I also like the simplicity, on the other hand it creates more issues than it solves, sometimes you can be too simple.

I'm actually somewhat OK with agility contributing to Supernatural attacks, because you could argue that it's possible to avoid the magic energy. Though I'd prefer if some effects couldn't be dodged.

What I have a huge problem with is Armor contributing to defense against those effects though, specifically poison and disease, it feels weird. The new Toughness would be a way to bypass armor and deal with the Plated Tank Issue. Without it you need to nerf armor, which makes it loose value and feel less impactfull.

You also muddle the difference between the various character builds, if you defend against everything the same, attacks become sam-ish and no one has the chance to shine. Before you had the Paladin who could go 1 on 1 with the Werewolf and the Mage, who could deal with the Barbarian. Some of that would be gone and it'd feel worse for everyone.

Having Fortitude affect three defenses seems like a much smaller issue. 

However if you want to get rid of that how about removing Fortitude from Evasion like istabosz suggested but actually **increase** the value of all armors to keep the stat effective. Fortitude would still play a role anyway, because it contributes to HP, and with better armors for both Agility and Might, you wouldn't need as much Fortitude.

I think that way would provide both more options and more balance. 

An Agility character could get some light armors and a Might character could get heavy armor, both would get decend defense values, but be vulnerable to certain types of magic unless they also have fortitude or presense. More mental/social focussed Characters could get more defense against certain Supernatural attacks, but be more vulnerable against direct attacks unless they also get fortitude. 

You could get a character that is focussed on defenses, but he would lack offense, or you could get one that is all physical defense and offense, but without mental, he'd lack HP and Resolve. and without Presence his Toughness wouldn't be that high.

You could change(keep) the name to Evasion (yes Eva**sion**, evade is a verb) and maybe change Toughness to Tenacity to account for the presence of Presence

So that whole thing would look like this:

HP: 10 + 2 x (Fortitude + Presence + Will) 
Tenacity: 10 + Fortitude + Presence
Evasion:  10 + Agility + Armor
Resolve: 10 + Presense + Will
Light Armor: 3-5 Defense req. 0-1 Might
Heavy Armor: 6-12 Defense req. 2-6 Might,  Agility penalties

Damn Wall O' Text, but I think It covers all angles now, but then it's my essay, so poke away! (;

**Comment by raihje** [comment]
I can't really put my finger on it, but having armour only contribute to Evasion doesn't sit right with me. I liked Brians original proposal to have a basic defense stat, then your fortitude and resolve for different effects, banes, etc. Will and Fortitude both contribute to HP so that'd be their secondary priority - meanwhile most attempts to hit a person would be through their defensive stat Whether we call that resilience or what have you.

Resilience and Resolve is a good push, agility can also represent your ability to stay on your feet despite the world spinning due to poison, etc. A simple physical/mental split is good but as others have said it might feel a bit "watered down". OpenLegend is already particularly simplified and abstracted so that gameplay is smooth, but if you make it too simplified you risk "sameness" of various encounters and perhaps a lack of meaningful specialization in certain areas.

**Comment by brianfeister** [comment]
@istabosz what do you think?

**Comment by istabosz** [comment]
Here's what I like best of the proposals so far:

Toughness = 10 + Fortitude + Presence
Evasion = 10 + Agility + Armor
Resolve = 10 + Presence + Will

1. I like keeping the name "Toughness" because it works well enough and tis better not to change terms if you don't need to.

2. I like that we've covered the 4 basic defenses/saves that D&D has, which I think work quite well. We've got a defense for poison/life drain/etc, one for domination/illusion/fear, and one for attacks/fireballs/rays. I think Evasion works quite well as a combo of both AC and Reflex saves, and I really like the fact that armor and shields will now help protect against fireballs but NOT poison gas.

3. I don't think that we need to limit agility based on armor type anymore because we no longer have two attributes contributing to Evasion.

4. I do like Might instead of Fortitude for armor prereq. as that evens up the power of Might a good deal and makes it less likely for someone to have uber Evasion.

**Comment by Hvalfisk** [comment]
Isn't Presence a little strong now? Like, strictly better than both Fortitude and Will, strong. 

Fortitude => 2HP + 1 Toughness
Will => 2HP + 1 Resolve
Presence => 2HP + 1 Toughness + 1 Resolve
Agility => Damage + Evasion
Might => Damage + Heavier Armor (Evasion)

¿ Maybe HP = 2x(Fortitude + Will) + 10 ? 

If a PC drinks a slightly poisoned ale, would you ask them to make a Fortitude roll against a CR of 15?

**Comment by brianfeister** [comment]
@Hvalfisk Nope, Fortitude factors into HP but Presence does not.

I'm good with @istabosz proposal, it's interesting that Evasion is going to become VERY useful since we will be funneling all physical / melee attacks toward it. But I consider that acceptable. And the value in NOT swapping out all of the existing defenses I believe is HUGE.

Thank you everyone for all of your time and effort in helping to hammer out a solution!

Let's put this in the queue for pending rules change. I'm locking this thread now and calling the decision final.

**Comment by istabosz** [comment]
> Nope, Fortitude factors into HP but Presence does not.

Are we changing the HP formula? Currently it is 2 x (Fortitude + Presence + Will ) + 10

**Comment by brianfeister** [comment]
Whoops. My mistake. I'm good with the change. I suppose The point is that Fortitude and Presence are defensive only. I don't know if we care about Presence having more impact

**Comment by istabosz** [comment]
Ok. Sounds good to me. I'm going to update the rules

**Comment by istabosz** [comment]
So, we've also got to edit the rules in the combat chapter a bit.

Do area effects now target evasion?

**Comment by istabosz** [comment]
Maybe we should just say that supernatural attacks target the most logical defense. So, a fireball would target evasion, but a poison gas cloud targets toughness.

**Comment by brianfeister** [comment]
Ugh. That's annoying because it could also be Evasion to get out of the way of the gas cloud.

**Comment by istabosz** [comment]
Well, armor isn't gonna help you from the gas. Plus, at the end of the day, whatever call we make isn't gonna be 100% realistic. I think that using toughness against poisony / life drainy attacks and evasion against dodgey / explosiony attacks will mesh with what 99% of gamers are used to.

**Comment by brianfeister** [comment]
That's fine. Caveat it as GMs discretion and let's get 'er done

**Comment by brianfeister** [comment]
@istabosz waiting on you to update the rules

**Comment by istabosz** [comment]
This is done and the changes have been pushed to the website, so I'm closing.

**Comment by brianfeister** [comment]
MUAHAHAHA!!! You thought that this thread was dead, but NAY, IT LIVES!

Why did we settle on `Toughness = Fort + Presence`… shouldn’t it be `Fort + Will`? 

And as far as I can tell, we have to choose one attribute to get the short end of the stick (both factor into HP + one other defense, Resolve).

If either `Will` or `Presence` are getting the short end of the stick in terms of mechanical impact then given that Presence has a bunch of boons attached to it, seems to me that Will should win out here. 

Also I feel like Toughness is **obviously** a combination of your `Will + Fort` isn't it? Toughness isn't related to `Presence` because there's no "luck factor", right?

**Comment by istabosz** [comment]
Agreed. Toughness should be Fortitude + Will

**Comment by istabosz** [comment]
@brianfeister Is this issue resolved? Looks like it's been updated in the rules

**Comment by brianfeister** [comment]
yep!
